docs/site/HasMany-relation.md

```diff
@@ -513,7 +513,7 @@ allows users to retrieve all customers along with their related orders through
 the following code at the repository level:
 
 ```ts
-customerRepo.find({include: [{relation: 'orders'}]});
+customerRepo.find({include: ['orders']});
```

Comment:
Let's update the controller API with the simplified form in the rest of the file as well.